silverstripe/framework/CVE-2020-26138.yaml

```diff
@@ -0,0 +1,11 @@
+title:     'CVE-2020-26138 FormField: with square brackets in field name skips validation'
+link:      https://www.silverstripe.org/download/security-releases/cve-2020-26138
+cve:       CVE-2020-26138
+branches:
+    3.0.x:
+        time:     2021-06-07 22:31:00
+        versions: ['>=3.0.0', '<3.99.99']
```

Comment:
this should be `<4.0.0` as there is no patched 3.99.99 version.

Reply:
```suggestion
        versions: ['>=3.0.0', '<4.0.0']
```

Reply:
I've updated to <4.0.0